tools/migrate-devspaces-assets.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+set -e
+
+# Please install the below pre-requisites if using this on your local machine, or alternately, you can just use azure bash cloudshell for a seamless experience.
+#1. azure cli    :   https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest
+#2. kubectl      :   https://kubernetes.io/docs/tasks/tools/install-kubectl/
+#3. docker       :   https://docs.docker.com/engine/install/
+#4. curl
+#5. gunzip
+#6. tar
+
+HELMDIR=/var/tmp/helm_migrate
+INGRESSNAME=migration-traefik
+ENABLEINGRESS=false
+
+echo ""
+echo "Migrate assets of Azure Dev Spaces to Bridge to Kubernetes"
+echo "----------------------------------------------------------"
+echo ""
+
+helpFunction()
+{
+   echo ""
+   echo "Usage: $1 -g ResourceGroupName -n AKSName -h DockerHubName"
+   echo -e "\t-g Name of resource group of AKS Cluster"
+   echo -e "\t-n Name of AKS Cluster"
+   echo -e "\t-k Kubernetes namespace (uses 'default' namespace otherwise)"
+   echo -e "\t-r Path to Root of the project that needs to be migrated (default = pwd)"
+   echo -e "\t-h Docker Hub repository name"
+   echo -e "\t-t Docker image name & tag in format name:tag (default = projectName:stable)"
+   echo -e "\t-i Enable ingress (default = false)"
+   echo -e "\t-d Helm Debug switch"
+   exit 1 # Exit script after printing help
+}
+
+installHelmFunction()
```

Comment:
In case we see any errors here, we should provide a link to install manually and ask users to run this command again passing the location as an argument